api/transaction/routes.go

```diff
@@ -525,10 +541,19 @@ func ComputeTransactionGasLimit(c *gin.Context) {
 }
 
 func getQueryParamWithResults(c *gin.Context) (bool, error) {
-	withResultsStr := c.Request.URL.Query().Get("withResults")
+	withResultsStr := c.Request.URL.Query().Get(queryParamWithResults)
 	if withResultsStr == "" {
 		return false, nil
 	}
 
 	return strconv.ParseBool(withResultsStr)
 }
+
+func getQueryParameterCheckSignature(c *gin.Context) (bool, error) {
+	bypassSignatureStr := c.Request.URL.Query().Get(queryParamCheckSignature)
+	if bypassSignatureStr == "" {
+		return true, nil
```

Comment:
is this intended behavior?
I think that now if the query param is missing, then it would be `true` as default

Reply:
please refactor: use everywhere in this PR `bypassSignatureCheck` instead of `checkSignature`. So adjust the logic. if `bypassSignatureCheck` then use a disabled sig verifier, and so on. It is misleading right now. Checking signature is the default option, so bypassing it should be activated by a bool flag..